cmd/verify/cmd.go

```diff
@@ -37,5 +40,10 @@ func init() {
 	Cmd.AddCommand(oc.Cmd)
 	Cmd.AddCommand(permissions.Cmd)
 	Cmd.AddCommand(quota.Cmd)
-	Cmd.AddCommand(rosa.Cmd)
+	r, err := rosa.VerifyRosaCmd()
```

Comment:
@ciaranRoche I think we can tidy this up by dropping the `error` from the signature of the `reporter.Builder.Build` function. Lemme take a look 👍